README.md

```diff
@@ -4,27 +4,121 @@ Translate is a library for machine translation written in PyTorch. It provides t
 
 ## Requirements and Installation
 
-Translate requires
+### Translate requires:
 * A Linux operating system with a CUDA compatible card
 * C++ compiler supporting ECMAScript syntax for `<regex>`, such as GCC 4.9 and above
 * A [CUDA installation](https://docs.nvidia.com/cuda/). We recommend CUDA 8 or CUDA 9
 
-To install Translate from source:
-- Install Anaconda or Miniconda3 from https://conda.io/miniconda.html:
-  ```
+### To install Translate from source:
+These instructions were mainly tested on CentOS 7.4.1708 with a Tesla M40 card
+and a CUDA 8 installation. We highly encourage you to [report an issue](https://github.com/pytorch/translate/issues)
+if you are unable to install this project for your specific configuration.
+
+- If you don't already have an existing [Anaconda](https://www.anaconda.com/download/)
+environment with Python 3.6, you can install one via [Miniconda3](https://conda.io/miniconda.html):
+  ```bash
   wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
   chmod +x miniconda.sh
   ./miniconda.sh -b -p ~/miniconda
   rm miniconda.sh
   . ~/miniconda/bin/activate
   ```
-- Please refer to the `install.sh` script. In many cases you can simply run `bash install.sh`. We have tested this script on CentOS 7.4.1708 with a Tesla M40 card and a CUDA 8 installation. We encourage you to report an [issue](https://github.com/pytorch/translate/issues) if you are unable to install this project for your specific configuration.
 
-Alternatively, you can launch an AWS instance using the `pytorch_translate_initial_release` image (AMI ID: ami-04ff53cdd573658dc). Once you have ssh'ed to the AWS instance, the example commands below should work after running `cd translate`.
+- Clone the Translate repo:
+  ```bash
+  git clone --recursive https://github.com/pytorch/translate.git
+  pushd translate
+  ```
+
+- Build [PyTorch](https://pytorch.org/) from source (currently needed for ONNX compatibility):
+  ```bash
+  # Uninstall previous versions of PyTorch. Doing this twice is intentional.
+  # Error messages about torch not being installed are benign.
+  pip uninstall -y torch
+  pip uninstall -y torch
+
+  # Install basic PyTorch dependencies.
+  conda install -y cffi cmake mkl mkl-include numpy pyyaml setuptools typing
+  # Add LAPACK support for the GPU.
+  conda install -y -c pytorch magma-cuda80 # or magma-cuda90 if CUDA 9
+
+  # Install NCCL2.
+  wget https://s3.amazonaws.com/pytorch/nccl_2.1.15-1%2Bcuda8.0_x86_64.txz
+  tar --no-same-owner -xvf nccl_2.1.15-1+cuda8.0_x86_64.txz
```

Comment:
Why do we have `--no-same-owner` option?

Reply:
Seems that may not be needed. Removing.